wordpress-mobile/gutenberg-mobile
# Implement quote block in RN

Refs #207 

Gutenberg PR: https://github.com/WordPress/gutenberg/pull/15482

How to test:

 - Open the demo app and check if quotes in content are displayed correctly
 - Interact with quote elements by adding text, new lines on the quote part and citation part.
 - Try adding new quote element
 - Write on it.
